app/views/admin/works/_full_text.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="card bg-light mb-3">
+  <h2 class="card-header h3">Full text
+  </h2>
+  <div class="card-body">
+    <div class="text-muted small mb-2">The full text file is the transcript we use for full-text indexing.</div>
```

Comment:
Le'ts not call it "the full text file", that doesn't mean a lot. Can we call it "Searchable Transcript Text"?  

Can we also specify something like "This will be used for searchable transcripts only if there is NOT an OHMS XML transcript. If there is an OHMS XML transcript, this file will not be used.", for clarity?

Reply:
Yup. Added verbiage to that effect. Good call.